src/components/ListOfGoods/ListOfGoods.tsx

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+
+type Props = {
+  goods: string[];
+};
+
+export const ListOfGoods: React.FC<Props> = ({ goods }) => (
+  <ul>
+    {goods.map(good => (
+      <li
+        data-cy="Good"
+        key={good}
+      >
+        {good}
+      </li>
```

Comment:
looks good but add color to li`s

Reply:
Sorry, can you please specify, i don't have any color inside 'good', where do I get the color from?